modules/cli/src/test/scala/cli/tests/LauncherCliTest.scala

```diff
@@ -19,4 +20,18 @@ class LauncherCliTest extends munit.FunSuite {
     expect(nightlyCliVersion.endsWith("-SNAPSHOT"))
   }
 
+  val expectedScalaCliVersions = Seq(
+    "0.1.2"                       -> Constants.defaultScala212Version,
+    "0.1.1+43-g15666b67-SNAPSHOT" -> Constants.defaultScala212Version,
+    "0.1.3"                       -> Constants.defaultScala213Version,
+    "nightly"                     -> Properties.versionNumberString
+  )
+
+  for ((cliVersion, expectedScalaVersion) <- expectedScalaCliVersions)
+    test(s"use expected scala version for JVM Scala CLi launcher: $cliVersion") {
```

Comment:
NIT typo + remove `JVM`
```suggestion
    test(s"use expected scala version for Scala CLI launcher: $cliVersion") {
```